src/common/BaseAnnotator.ts

```diff
@@ -160,6 +163,11 @@ export default class BaseAnnotator extends EventEmitter {
         return typeof selector === 'string' ? document.querySelector(selector) : selector;
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    protected handleChangeMode({ mode }: { mode: store.Mode }): void {
```

Comment:
Individual annotators should add this if they need it rather than relying on indirect event handling via the base class.